### ISSUE TITLE
Consider GitHub Issue Comments instead of Disqus

### ISSUE THREAD

**Issue by iAmBipinPaul** [statiqdev/Statiq#755]
Blog Recipe comes with the Disqus. It will be better if it will come with GitHub Issue Comments. As we know the people who will be using #Wyam must be already familiar with GitHub as well as readers also.

Option.
[Utterance : ](https://utteranc.es/) A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!

 Utterance  is built by Jeremy Danyow. Jeremy works as a Senior Software Engineer at Microsoft, he is member of the Aurelia core team and created also [gist.run.](https://gist.run/)

Official site : https://utteranc.es/
Repo : https://github.com/utterance
Relates blogs : https://asp.net-hacker.rocks/2018/11/19/github-comments.html

**Comment by daveaglick** [comment]
I totally agree - I've actually been wanting to get rid of Disqus on my own blog for a while but just haven't gotten around to it. I think an even better option would be to make the comment system swappable, but that may take a little thinking on how to do it right.

**Comment by iambipinpaul** [comment]
I just replaced Disqus with GitHub issue comments and I  just had to remove and Disqus script and replace that with in  ```_PostFooter.cshtml``` file.
```
<meta property="og:title" content="@Model.String(BlogKeys.Title)"/>
<script src="https://utteranc.es/client.js"
        repo="iambipinpaul/bipinpaul.com.np"
        issue-term="og:title"
        theme="github-light"
        crossorigin="anonymous"
        async>
</script>
```

**Comment by arebee** [comment]
This depends entirely on the template. Would be better IMHO to add both and comment them both out and let people choose by modifying the template. KISS.
